src/components/pages/Oversikt/Oversikt.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { Undertittel } from 'nav-frontend-typografi';
+import { Panel } from 'nav-frontend-paneler';
+import { withBehandlingContext } from '../../../context/BehandlingerContext';
+import { oversikttekster } from '../../../tekster';
+import { toDate } from '../../../utils/date';
+import './Oversikt.less';
+import { withRouter } from 'react-router';
+import Lenke from 'nav-frontend-lenker';
+
+const Beregning = withBehandlingContext(
+    ({ behandlinger, fetchAlleBehandlinger, setValgtBehandling, history }) => {
+        useEffect(() => {
+            fetchAlleBehandlinger();
+        }, []);
+
+        const velgBehandling = behandling => {
+            setValgtBehandling(behandling);
+            history.push('/sykdomsvilkår');
+        };
+
+        return (
+            <Panel border className="Oversikt">
+                <Undertittel className="panel-tittel">{oversikttekster('tittel')}</Undertittel>
+
+                <table className="tabell">
+                    <thead>
+                        <tr>
+                            <th>{oversikttekster('søker')}</th>
+                            <th>{oversikttekster('periode')}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {behandlinger.map(behandling => (
+                            <tr key={behandling.behandlingsId}>
+                                <td>
+                                    <Lenke onClick={() => velgBehandling(behandling)}>
+                                        {behandling.personinfo.navn
```

Comment:
`behandling.personinfo` kan være undefined her, hvis man er på oversikt-siden og søker opp en behandling. Det skyldes at etter at søket er gjort så blir `setBehandlinger` kalt (i `fetchBehandlinger` i `BehandlingerContext`) og derfor blir Oversikt-komponenten rendret på nytt, uten at personinfo har blitt hentet. Forslag til løsning er å legge på en "?" 😄 (pluss ternary-en kan erstattes med short circuiting):
```jsx
<Lenke onClick={() => velgBehandling(behandling)}>
    {behandling.personinfo?.navn ||
        behandling.originalSøknad.aktorId}
</Lenke>
```

Reply:
Godt sett! Der var jeg litt for kjapp.